buildscript/modules/clang.cmd

```diff
@@ -16,13 +17,17 @@
 @if /I "%useclang%"=="y" IF %toolchain%==msvc IF EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set PATH=%llvminstloc%\%abi%\bin\;%PATH%
 @if /I "%useclang%"=="y" IF %toolchain%==msvc IF EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set llvmalreadyloaded=1
 @if /I "%useclang%"=="y" IF %toolchain%==msvc IF %abi%==x64 IF NOT EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set PATH=%ProgramFiles%\LLVM\bin\;%PATH%
+@if /I "%useclang%"=="y" IF %toolchain%==msvc IF %abi%==aarch64 IF NOT EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set PATH=%ProgramFiles%\LLVM\bin\;%PATH%
 @if /I "%useclang%"=="y" IF %toolchain%==msvc IF %abi%==x86 IF /I %PROCESSOR_ARCHITECTURE%==AMD64 IF NOT EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set PATH=%ProgramFiles% (x86)\LLVM\bin\;%PATH%
 @if /I "%useclang%"=="y" IF %toolchain%==msvc IF %abi%==x86 IF /I %PROCESSOR_ARCHITECTURE%==x86 IF NOT EXIST "%llvminstloc%\%abi%\bin\clang-cl.exe" set PATH=%ProgramFiles%\LLVM\bin\;%PATH%
 @if /I "%useclang%"=="y" IF %toolchain%==msvc set CC=clang-cl.exe
 @if /I "%useclang%"=="y" IF %toolchain%==msvc set CXX=clang-cl.exe
-@if /I "%useclang%"=="y" IF NOT %toolchain%==msvc set MSYSTEM=CLANG%MSYSTEM:~-2%
-@if /I "%useclang%"=="y" IF NOT %toolchain%==msvc set LMSYSTEM=clang%MSYSTEM:~-2%
+@rem These two lines don't work on arm64 becuase they are 5 chars and it only takes the last 2 chars 
+@rem @if /I "%useclang%"=="y" IF NOT %toolchain%==msvc set MSYSTEM=CLANG%MSYSTEM:~-2%
+@rem @if /I "%useclang%"=="y" IF NOT %toolchain%==msvc set LMSYSTEM=clang%MSYSTEM:~-2%
```

Comment:
Not sure since on MSVC, there's ARM64 and [Arm64EC](https://learn.microsoft.com/en-us/windows/arm/arm64ec). Both are for ARM64 CPUs but are different ABIs to compile for.

Reply:
I don't think Clang supports Arm64EC so it's probably not an issue for now, at least.